README.md

```diff
@@ -1,2 +1,211 @@
-# Hello_Microverse
-My first repository using Git &amp; GitHub
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ ] **[OPTIONAL]** You can remove all this commented code to make your README file look more professional and organized. 🎯

Reply:
Ok, perfect. Thank you very much for your support!